packages/yourloops/lib/auth/api.ts

```diff
@@ -506,7 +506,7 @@ async function updateSettings(auth: Readonly<Session>): Promise<Settings> {
 }
 
 async function updateUser(auth: Readonly<Session>, updates: Partial<User>): Promise<void> {
-  const updateURL = new URL("/auth/user", appConfig.API_HOST);
+  const updateURL = new URL(`/auth/user/${auth.user.userid}`, appConfig.API_HOST);
```

Comment:
The change role has been fixed in https://github.com/mdblp/blip/pull/499 differently

Reply:
cool it is fine. i will remove the change.